lib/config.js

```diff
@@ -400,6 +403,7 @@ function normalize (opts, logger) {
   normalizeNumbers(opts)
   normalizeBytes(opts)
   normalizeArrays(opts)
+  normalizePositiveTime(opts, logger)
```

Comment:
👍 I ran some tests and this function appears to behave as expected -- non-positive and null/undefined values use the values from DEFAULTS.